### PR TITLE
Sk 25

### PR DESCRIPTION
**Pull request by jhunnie**
Right now, waiting on Russell's code for actual logo url and adding that to the model. Code creates a PNG image and saves a temporary file to whatever local directory you're in. I can create another folder just for these temporary images if necessary. The function that will store the image to S3 and generate the image_url will be needed before I delete the image from local. 

Testing: just some basic local testing with various string lengths to check for any writing overflow.

### CODE REVIEW

File: postings/models.py [hunk 1]
```diff
@@ -63,20 +70,67 @@ def generate_image(self):
         # title = self.tile to access parameters
         # each posting has company logo, job title, experience level, location, and posting link
         #img = Image.new('RGB', (570, 430), color = (255, 255, 255))
```
Comment (wangwalton) [inline postings/models.py]: rm

File: postings/models.py [hunk 2]
```diff
@@ -89,8 +143,9 @@ def save_image(self, image):
...
         """
+
+
         pass
 
     def __str__(self):
-        return '{:39}{:24}{:24}{}\t{:70}'.format(self.title[:35], self.company[:20],
-                self.city[:20], self.date_posted, self.url[:70])
\ No newline at end of file
+        return "%s %s %s" % (self.company, self.title, self.date_posted)
```
Comment (wangwalton) [inline postings/models.py]: revert

File: postings/models.py [hunk 3]
```diff
@@ -63,20 +70,67 @@ def generate_image(self):
...
+        # creating default margin sizes 
+        left_margin, right_margin, top_margin, bottom_margin = 30,30,30,30
+        spacing = 30
+        line_space = 40
+        
+        # create variable to draw on the image and add text 
+        d = ImageDraw.Draw(background)
+
+        # Drawing text for job posting information 
+        # draw text, Company Title
```
Comment (wangwalton) [inline postings/models.py]: rm

**Review by wangwalton — CHANGES_REQUESTED** [review]
Can you upload 3 examples of what the card looks like?

File: postings/models.py (hunk 1 above)
Comment (jhunnie) [reply postings/models.py]: done

File: postings/models.py (hunk 2 above)
Comment (jhunnie) [reply postings/models.py]: done

**Pull request merged**